modules/UI/UI.js

```diff
@@ -1403,10 +1391,7 @@ UI.hideRingOverLay = function () {
  * @returns {*|boolean} {true} if the overlay is visible, {false} otherwise
  */
 UI.isOverlayVisible = function () {
-    return RingOverlay.isVisible()
-        || SuspendedOverlay.isVisible()
-        || PageReloadOverlay.isVisible()
-        || GumPermissionsOverlay.isVisible();
+    return RingOverlay.isVisible() || this.overlayVisible;
```

Comment:
Eventually it should be. The thing is that it was not extending Overlay.js and the refactoring will be a little bit harder that the other overlays.